locale-more-styles/ckb/long-time.json

```diff
@@ -0,0 +1,30 @@
+{
+	"year": {
+		"one": "{0} ساڵ",
```

Comment:
I can see that `one` is equal to `other`.
It could just be written as `"year": "{0} ساڵ"` in that case.
And for other time measurement units too, where applicable.

Reply:
thanks for your input 🙏 I've deleted the redundant values.